### PR TITLE
Remove support for Mobile Websites

### PR DESCRIPTION
**Pull request by teobugslayer**
`cloud export` should continue working.
This PR should be merged post-3.0 release after Web Sites support is toggled off for all users.

### CODE REVIEW

File: lib/services/samples-service.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export class SamplesService implements ISamplesService {
 		return (() => {
 			this.$logger.info("You can choose a sample from the following: %s", EOL);
 			if(framework) {
+				if (framework)
```
Comment (rosen-vladimirov) [inline lib/services/samples-service.ts]: hm, I'm pretty sure one if is enough for this check :smile_cat:

**Comment by rosen-vladimirov** [comment]
`dynamicHelpProvider` defines `isMobileWebsite` variable for our help, I believe we should remove it as well

**Comment by teobugslayer** [comment]
We need to leave isMobileWebsite  defined. We still support `cloud export` and `publish` commands and their help must be showable (if that is a word).

**Comment by teobugslayer** [comment]
run ci

File: lib/project/web-site-project.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export class MobileWebSiteProject extends frameworkProjectBaseLib.FrameworkProje
 	}
 
 	public get name(): string {
-		return this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite;
+		throw new Error("Тhe development of mobile websites is discontinued. For more information, see http://www.telerik.com/blogs/whats-next-for-telerik-platform-and-web-projects");
```
Comment (Fatme) [inline lib/project/web-site-project.ts]: `this.$errors.failWithoutHelp` maybe. Also you extract the message to separate variable/function.

**Comment by teobugslayer** [comment]
run ci

**Comment by teobugslayer** [comment]
Will retarteg against master

**Pull request closed without merging**